test/iterator.js

```diff
@@ -0,0 +1,239 @@
+'use strict';
+
+const { Iterator } = api.common;
+
+const array = [1, 2, 3, 4];
+
+const test = (name, test) => api.metatests.test(name, test);
+
+test('Iterator is iterable', test => {
+  const iterator = new Iterator(array);
```

Comment:
I assume we don't have, `beforeEach`, `afterEach` functionality in metatests? =(

Reply:
We have no